settings/DevHome.Settings/Views/AccountsPage.xaml.cs

```diff
@@ -84,7 +103,24 @@ private async void AddDeveloperId_Click(object sender, RoutedEventArgs e)
         {
             if (addAccountButton.Tag is AccountsProviderViewModel accountProvider)
             {
-                await ShowLoginUIAsync("Settings", this, accountProvider);
+                if (accountProvider.DeveloperIdProvider.GetAuthenticationExperienceKind() == Microsoft.Windows.DevHome.SDK.AuthenticationExperienceKind.CardSession)

```

Comment:
May be a good idea to add a using statement for this Microsoft.Windows.DevHome.SDK namespace.

Reply:
Addressed in recent commit. Thanks!